index.bs

```diff
@@ -180,6 +180,30 @@ by showing the user that certain things won't happen without their permission.
 But frequently asking users for consent can also show how scary a place the web is
 by showing how many sites are willing to ask for intrusive and unnecessary permissions.
 
+<h3 id="devices-platforms">Support the full range of devices and platforms</h3>
+
```

Comment:
Please add a `<!-- was "Media Independence" in the HTML Design Principles -→` comment here. I've been doing this for principles which get ported over. (See for instance #190.)

Reply:
Done in 717bc8d.